Simplify quickstart by removing unneeded step (#2984)
pipe-cd/pipe: docs/content/en/docs-dev/quickstart/_index.md

  This page describes how to quickly get started with PipeCD on Kubernetes.
---

> This guides you to install PipeCD in your kubernetes and deploy a "hello world" application to that Kubernetes cluster. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).

Reviewer: ```suggestion
This page is a guideline for installing PipeCD in your Kubernetes cluster and deploying a "hello world" application to that same Kubernetes cluster.

Note that it __does not require installing PipeCD to your all clusters to enable using PipeCD to deploy your applications__, we do it here just for the simplicity of this guideline. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
```
nits

  This page describes how to quickly get started with PipeCD on Kubernetes.
---

This page is a guideline for installing PipeCD in your Kubernetes cluster and deploying a "hello world" application to that same Kubernetes cluster.

> Note that it __does not require installing PipeCD to your all clusters to enable using PipeCD to deploy your applications__, we do it here just for the simplicity of this guideline. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).

Reviewer 2: nit: It took me a little time to get it.
```suggestion
Note that it __does not require installing PipeCD to all clusters where you deploy your applications typically__, we do it here just for the simplicity of this guideline. For further reading about PipeCD's ideas, please visit [overview](/docs/overview/) and [concepts](/docs/concepts/).
```

Reviewer: That's better 👍

``` console
helm repo add pipecd https://charts.pipecd.dev

helm install pipecd pipecd/pipecd -n pipecd \
  --create-namespace \
  --dependency-update \
  --values https://raw.githubusercontent.com/pipe-cd/manifests/{{< blocks/latest_version >}}/quickstart/control-plane-values.yaml
```

Once installed, use `kubectl port-forward` to expose the web console on your localhost:

``` console
kubectl -n pipecd port-forward svc/pipecd 8080
```

> Then point your web browser to [http://localhost:8080](http://localhost:8080) to login with the configured static admin account as below:

Reviewer: ```suggestion
The PipeCD web console will be available at [http://localhost:8080](http://localhost:8080). To login, you can use the configured static admin account as below:
```
nits
